stream-chat-android-ui-components-sample/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Stream Chat Sample</string>
+    <string name="app_name">ChatSample</string>
```

Comment:
While we're changing things here, it might be a good idea to somehow differentiate between the old and the new apps in the launcher

Reply:
Yes, good idea. No need to provide a separate icon, but a different name would make our lives a little bit easier.

Reply:
We decided to use the same name for now